lib/broccoli/default-packager.js

```diff
@@ -44,12 +46,55 @@ module.exports = class DefaultPackager {
   packageBower(tree, bowerDirectory) {
     if (this._cachedBower === null) {
       this._cachedBower = new Funnel(tree, {
-        srcDir: '/',
```

Comment:
seems unrelated?

Reply:
it should have been removed in the first place, and I forgot to do it; unnecessary parameter